examples/powercli/ha-restarted-vms/README.md

```diff
@@ -0,0 +1,91 @@
+# HA Restarted VMs Email Notification
+
+## Description
+
+This function demonstrates using PowerShell and PowerCLI to send an HTML formatted email notification containing a list of all VMs which were restarted due to a host failure in an HA enabled cluster.  The body of the email contains the VM names, time the VMs were restarted, and the contents of the VM notes/description field.
+
+## Consume Function Instruction
+
+Step 1 - Clone repo
+
+```
+git clone https://github.com/vmware-samples/vcenter-event-broker-appliance
+cd vcenter-event-broker-appliance/examples/powercli/ha-restarted-vms
+git checkout master
+```
+
+Step 2 - Update `stack.yml` and `vcconfig-ha-restarted-vms.json` with your environment information
+
+<B>Note:</B> Be sure to configure all of the appropriate SMTP fields such as the SMTP_SERVER and SMTP_PORT for your particular SMTP server.  If authentication is not needed for your SMTP server, you can leave SMTP_USERNAME and SMTP_PASSWORD blank.
+
+Step 3 - Login to the OpenFaaS gateway on vCenter Event Broker Appliance
+
+```
+VEBA_GATEWAY=https://veba.abc.com
+export OPENFAAS_URL=${VEBA_GATEWAY}
+
+faas-cli login --username admin --password-stdin --tls-no-verify
+```
+
+Step 4 - Create function secret (only required once)
+
+```
+faas-cli secret create vcconfig-ha-restarted-vms --from-file=vcconfig-ha-restarted-vms.json --tls-no-verify
+```
+
+Step 5 - Deploy function to vCenter Event Broker Appliance
+
+```
+faas-cli deploy -f stack.yml --tls-no-verify
+```
+
+## Build Function Instruction
+
+Step 1 - Clone repo
+
+```
+git clone https://github.com/vmware-samples/vcenter-event-broker-appliance
+cd vcenter-event-broker-appliance/examples/powercli/ha-restarted-vms
+git checkout master
+```
+
+Step 2 - Initialize function, only required during the first deployment
+
+```
+faas-cli template pull
```

Comment:
I heard from @rguske that this command actually does not pull the `powercli` template anymore which is used in your function. Is that correct? If so, we need to update this `README` since otherwise the build would not work as described.